### DISCUSSION TITLE
In the latest version of ariakit (alpha-19) there is no ariakit/select, how come?

### DISCUSSION THREAD

**Discussion by devchn** [ariakit/ariakit#1111] [Questions] [answered]
<img width="930" alt="image" src="https://user-images.githubusercontent.com/22728668/158537351-3372ade1-4eb7-47df-b816-d0208f4f5c6a.png">
<img width="1046" alt="image" src="https://user-images.githubusercontent.com/22728668/158537427-d6ef9455-77bb-458b-8321-c07155ea4d8e.png">
<img width="411" alt="image" src="https://user-images.githubusercontent.com/22728668/158537983-aea1b20c-4b61-4f0d-98c5-3fba379daecc.png">

In the latest version of ariakit (alpha-19) there is no ariakit/select, how come?

**Comment by diegohaz** [comment, marked as answer]
It's not released yet. It'll be released on alpha.20 (likely next week).
